src/Controller/Speaker/Create.php

```diff
@@ -48,7 +48,13 @@ public function handle(Request $request): Response
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            $speakerRequest->photoPath = $this->fileUploader->upload($speakerRequest->photo);
+            if (!empty($speakerRequest->photo)) {
+                $speakerRequest->photoPath = $this->fileUploader->upload($speakerRequest->photo);
+            }
+
+            if (empty($speakerRequest->photoPath)) {
+                $speakerRequest->photoPath = 'http://'.$_SERVER['HTTP_HOST'].'/images/default_speaker.svg';
```

Comment:
Don't use hardcoded protocol. If I install the application using https, users will have a warning about mixed content

Reply:
And use `UrlGeneratorInterface` as discussed, I gave you the link to https://symfony.com/doc/current/routing.html#generating-absolute-urls for a reason

Reply:
Also, use the `Packages` service from Symfony assets to get the path, in case assets versioning has been applied.